functions/icons.zsh

```diff
@@ -86,6 +87,76 @@ case $POWERLEVEL9K_MODE in
   'awesome-fontconfig')
     # fontconfig with awesome-font required! See
     # https://github.com/gabrielelana/awesome-terminal-fonts
+    # if not defined, set recommended linux path
+    typeset -p "POWERLEVEL9K_FONTAWESOME_PATH" > /dev/null 2>&1  || POWERLEVEL9K_FONTAWESOME_PATH=~/.fonts
```

Comment:
Btw. the docs are already updated: https://github.com/gabrielelana/awesome-terminal-fonts/commit/1397337e349c8791b64088995bfba820784aed18
Thx. @gabrielelana 👍